lib/config-util.js

```diff
@@ -40,7 +40,11 @@ function loadConfig (argv, cwd) {
 // that would cause the application to exit early.
 Config.buildYargs = function (cwd) {
   cwd = guessCWD(cwd)
-  const config = loadConfig()
+  const argv = Yargs(process.argv)
+    .help(false)
```

Comment:
I think we could avoid needing to call `Yargs(process.argv)` here if we delay calling this line:

https://github.com/igorlima/nyc/blob/486aacc1ba1702c5f10505664cdefaa7368703ef/lib/config-util.js#L220

I think we could then swap out the config path in the `nyc.js` bin:

https://github.com/igorlima/nyc/blob/486aacc1ba1702c5f10505664cdefaa7368703ef/bin/nyc.js#L21

Reply:
@bcoe thanks for the quick _feedback_.

Done. Addressed already at https://github.com/istanbuljs/nyc/pull/724/commits/b01a3f34884ad3fb6c1a73b200ee7e0000443c6a.

- [x] avoid needing to call `Yargs(process.argv)`
- [x] swap out the config path in the `bin/nyc.js`

> _Let me know if there's any other concern._